my-tweet/src/components/userProfile.js

```diff
@@ -0,0 +1,56 @@
+import { Box,Avatar, Heading, Flex, Link,Button,useDisclosure, Wrap } from "@chakra-ui/react"
+import { useEffect,useRef } from "react"
+import { usePost } from "../context/post-context"
+import {NavLink} from 'react-router-dom'
+import { EditProfileModal } from "./edit-profile-modal"
+
+export const UserProfileComponent=()=>{
+    const { isOpen, onOpen, onClose } = useDisclosure()
+    const {loggedInUser,setLoggedInUser,userFollow}=usePost()
+
+    const getUserData=()=>{
+        const userData=JSON.parse(localStorage.getItem("userName"))
+        setLoggedInUser(userData)
+        console.log(userData,"userData");
```

Comment:
This console statement is to be removed.

Reply:
done